content/en/integrations/guide/azure-programmatic-management.md

```diff
@@ -12,17 +12,15 @@ further_reading:
+**All sites**: All [Datadog sites][3] can use the steps on this page to complete the App Registration credential process for Azure metric collection and the Event Hub setup for sending Azure Platform Logs.
 
-**US3**: If your organization is on the Datadog US3 site, you can use the Azure Native integration to streamline management and data collection for your Azure environment. Datadog recommends using this method when possible. Setup entails creating a [Datadog resource in Azure][14] to link your Azure subscriptions to your Datadog organization. This replaces the app registration credential process for metric collection and Event Hub setup for log forwarding.
+**US3**: If your organization is on the Datadog US3 site, you can use the Azure Native integration to streamline management and data collection for your Azure environment. Datadog recommends using this method when possible. Setup entails creating a [Datadog resource in Azure][14] to link your Azure subscriptions to your Datadog organization. This replaces the app registration credential process for metric collection and Event Hub setup for log forwarding. See the [Managing the Azure Native Integration][1] guide for more information.
 
 ## Datadog Azure integration
 
-The standard Azure integration process uses an app registration credential process for implementing metric collection, and an Event Hub setup for sending Azure Platform Logs. If your Datadog account is hosted on Datadog's [US3 site][3], you can use the [Datadog resource in Azure][14] to create and manage the Datadog integration with your Azure environment. This is referred to as the Azure Native integration, and enables you to configure log and metric collection, deploy the Datadog Agent, and manage account settings. See [Managing the Azure Native Integration][4] for additional information.
+The standard Azure integration uses an app registration credential process for implementing metric collection, and an Azure Event Hub setup for sending Azure Platform Logs. Create the app registration in Azure before integrating Datadog with your Azure environment, and configure it with the **Monitoring Reader** permission for Datadog to monitor the provided scope (subscriptions or management groups). If you don't already have an app registration created, see [Integrating through the Azure Portal][6] or [Integrating through the Azure CLI][4] for setup instructions.
 
-If your Datadog account is hosted on any other [Datadog site][3], use the standard [Azure integration][5] to collect monitoring data from your Azure environment.
-
-If you prefer to use CLI, see the [Azure CLI for Datadog][2].
+**Note**: You can assign read permissions at the management group level when creating the app registration in Azure, to monitor multiple subscriptions and have new subscriptions in the management group automatically monitored.
```

Comment:
```suggestion
You can assign read permissions at the management group level when creating the app registration in Azure, to monitor multiple subscriptions and have new subscriptions in the management group automatically monitored.
```

Reply:
The note is intended to emphasize the point, is there a particular reason to suggest its removal?

Reply:
I find that with "You can..." sentences, you don't need the **Note:** as it is the start of a new sentence, feel free to disregard this suggestion!

Reply:
Thanks for clarifying, I'll keep the note for now to retain the emphasis, but I'll keep that feedback in mind going forward :bow: